play26-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/play/api/ConductRApplicationLoader.scala

```diff
@@ -14,9 +14,11 @@ import play.api.{ Application, ApplicationLoader, Configuration, Environment, Mo
  */
 class ConductRApplicationLoader extends ApplicationLoader {
   def load(context: ApplicationLoader.Context): Application = {
+    val existingConfig = context.initialConfiguration
```

Comment:
@gmethvin - can I please confirm if `context.initialConfiguration` contain the merged config, i.e. `reference.conf` + `reference-overrides.conf` + the config either `application.conf` or specified via `-Dconfig.resource` or `-Dconfig.file`?

Reply:
Yes, at least in a typical Play app configuration. We basically set it up so all the ways you can create a Play server call out to `Configuration.load`, which handles that logic, so it should be safe to rely on `reference-overrides.conf`.